client/src/App.js

```diff
@@ -22,7 +23,17 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <div>
+              {/* Replace this Div with your Routes */}
+            <Route exact path="/" component={MovieList} />
+            <Route  path="/movies/:id "/>
```

Comment:
Looks good - be careful of extra spaces in your code, though (second Route). You want to keep it consistent